libs/cart/guides/installation.md

```diff
@@ -0,0 +1,46 @@
+# Installation
+
+## Table of Contents
+
+- Package Managers
+
+  - [Installing with NPM](#installing-with-npm)
+  - [Installing with yarn](#installing-with-yarn)
+
+- Platforms
+
+  - [Daffodil's In-Memory Web API](#in-memory-web-api)
+  - [Magento](#magento)
+  - [Shopify](#shopify)
+
+## Installing with `npm`
+
+To install the cart library and its dependencies, use the following command in your terminal.
+
+```bash
+npm install @daffodil/cart @daffodil/core @ngrx/store @ngrx/effects --save
+```
+
+## Installing with `yarn`
+
+To install the cart library and its dependencies, use the following command in your terminal.
+
+```bash
+yarn add @daffodil/cart @daffodil/core @ngrx/store @ngrx/effects
+```
+
+## Post-Install
```

Comment:
We should clarify what you're supposed to actually do post-install. Some people may only want `state` some people may only want `driver` and some people may only want a specific driver.

Reply:
I directed them to the getting started guide where I talk about the different ways to use Daffodil